### ISSUE TITLE
About Result

### ISSUE THREAD

**Issue by GG-nome** [filaPro/tr3d#7]
Hello, TR3D is an excellent model. However, when I tried to reproduce it, I found that SUN RGB-D could achieve results comparable to those in the paper. But on ScanNet, there was a 3% difference. The mAP@0.25 I obtained was 70.7, and the mAP@0.5 was 55.9. I used a GPU of 4090, Python 3.8 + CU11.7. The download and preprocessing were done according to MMdetection3D. The test was conducted using the test script of mmdetection3d. Could you please tell me what other issues might have been overlooked that prevented me from reproducing the results of the paper?

**Comment by filaPro** [comment]
i think it is just some randomness. you can train 5 times and 1 time it will be good. you can check the model and log file with good metrics here https://github.com/open-mmlab/mmdetection3d/tree/main/projects/TR3D

**Comment by GG-nome** [comment]
> i think it is just some randomness. you can train 5 times and 1 time it will be good. you can check the model and log file with good metrics here https://github.com/open-mmlab/mmdetection3d/tree/main/projects/TR3D

Thank you for your reply. May I ask if the results you reported for the Scannet dataset refer to the results submitted to the official server's Hidden Test Set or the results obtained from running on the local Val Set?

**Comment by filaPro** [comment]
i don't think test set supports evaluation of object detection (just instance segmentation), so we used validation set
